schema/bom-1.5.proto

```diff
@@ -604,6 +604,17 @@ message Vulnerability {
   repeated Property properties = 18;
   // The date and time (timestamp) when the vulnerability record was rejected (if applicable).
   optional google.protobuf.Timestamp rejected = 19;
+  // Evidence used to reproduce the vulnerability.
+  optional ProofOfConcept proofOfConcept = 20;
+}
+
+message ProofOfConcept {
+  // Precise steps to reproduce the vulnerability.
+  optional string reproductionSteps = 1;
```

Comment:
Thoughts on accepting workarounds and mitigation recommendations?

Reply:
recommendations are already part of the spec. Would workarounds be a separate field independent of recommendations?

Reply:
Yes, will be nice to have workarounds separately from mitigations.

Reply:
Workarounds have been added to the spec and test cases updated.